peps/pep-0667.rst

```diff
@@ -452,12 +453,18 @@ of both the frame and mapping proxy until the next cycle collection.
 
 ``PyEval_GetLocals()`` already creates a cycle, as it returns a borrowed reference.
 
+Implementation
+==============
+
+The implementation is in development as a draft pull request on GitHub ([2]_).
```

Comment:
We prefer inline links where there's no additional info listed in the references, it saves people a click.

```suggestion
The implementation is in development as a `draft pull request on GitHub
<https://github.com/python/cpython/pull/115153>`__.
```